.github/workflows/codeql.yml

```diff
@@ -0,0 +1,26 @@
+name: "Code Scanning"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  schedule:
+    - cron: '30 1 * * 0'
+
+jobs:
+  CodeQL-Build:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.0.2
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2.1.9
+        with:
+          languages: go
+      - name: Autobuild
+        uses: github/codeql-action/autobuild@v2.1.9
+      - name: Perform CodeQL Analysis
+        uses: github/codeql-action/analyze@v2.1.9
```

Comment:
These will need to reference the commit hash, ref https://github.com/actions-runner-controller/actions-runner-controller/pull/1422

Reply:
Thanks! Fixed in 854fbeaf9d50b99cbfcb0b3d75e0e84581c06327